docs/src/modules/utils/helpers.js

```diff
@@ -123,6 +123,10 @@ function getDependencies(raw, options = {}) {
     deps.typescript = 'latest';
   }
 
+  if (deps['@material-ui/lab'] && !deps['@material-ui/core']) {
+    deps['@material-ui/core'] = 'latest';
```

Comment:
Nice! Did you check the other packages for peer dependencies? It might make sense to abstract this into a generic `includePeerDependencies(deps)` so that it's obvious why this is required.

Reply:
I haven't checked. Agree. I will look at it.